source/includes/steps-copy-cli.yaml

```
@@ -0,0 +1,114 @@
+title: Log In to MongoDB Cloud
+ref: copy-app-cli-log-in-to-mongodb-cloud
+content: |
+  To configure your app with {+cli-bin+}, you must log in to MongoDB Cloud using
+  an :atlas:`API key </configure-api-access/#programmatic-api-keys>` scoped to the organization or
+  project that contains the app.
+
+  .. code-block:: bash
+
+     realm-cli login --api-key="<MongoDB Cloud Public API Key>" --private-api-key="<MongoDB Cloud Private API Key>"
+
+---
+title: Pull the Latest Version of Your App
+ref: copy-app-cli-pull-the-latest-version-of-your-app
+content: |
+  You'll need a local copy of the configuration files for the {+app+} you're copying.
+  Pull a local copy of the latest version of your app:
+
+  .. code-block:: bash
+
+     realm-cli pull --remote="<Your App ID>"
+
+  .. tip::
+
+     You can also download a copy of your application's configuration files from
+     the :guilabel:`Deployment > Export App` screen in the {+ui+}.
+
+---
+title: Create New App
+ref: copy-app-cli-create-new-app
+content: |
+  Create a new {+app+}. This is where you'll add the configuration files from 
+  the exported app in later steps. To create the app, run: 
+
+  .. code-block:: bash
+     
+     realm-cli app create
+
+  Follow the CLI prompts to complete app creation. This creates a new directory on 
+  your local machine with the new app's config files.
+
+---
+title: Copy App Configuration to New App
+ref: copy-app-cli-copy-exported-app-to-new-app
+content: |
+  Delete all the configuration files in the new app except for ``realm_config.json``.
+  You'll handle ``realm_config.json`` in the next step. 
+
+  Copy all the :ref:`configuration files and folders <app-configuration>` of your exported app
+  to the imported app folder except for the ``realm_config.json`` file.
```

Maybe switch `imported` to `new` here given the reframing of this page?

```suggestion
  to the new app folder except for the ``realm_config.json`` file.
```